lib/srfi/41.scm

```diff
@@ -1,73 +1,54 @@
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;; primitive layer adapted from SRFI 41 reference impl
-;; TODO: rewrite this in terms or R7RS lazy primitives
 
-(define-record-type Stream
-  (make-stream box)
+(define-record-type <stream>
```

Comment:
Please revert this - Chibi consistently uses the convention of `Class-Name` instead of `<class-name>`.

The motivation is largely because this is used in both write `{Class-Name fields ...}` and match
`(@ Class-Name fields)`, so it's shorter and looks cleaner without the angle brackets.